### PR TITLE
feature dashboard structure

### LINKED ISSUE #84: Create the dashboard structure
## description

Create the structure of the dashboard

## where

`src/pages/Dashboard`

## expected use

- sidebar need fixed at 300px
- the content should be relative `width: calc(100% - 300px)`

## preview

image

## figma

https://www.figma.com/file/YeJgCxq0y9yrK5Tk2r4ywF/desktop?node-id=340%3A0

### PR DESCRIPTION
**Pull request by diegocosta-dev**
resolves #84

### CODE REVIEW

File: src/pages/Dashboard/Dashboard.styles.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components';
 
-export const Calcs = styled.div`
+export const Operations = styled.div`
```
Comment (diegocosta-dev) [inline src/pages/Dashboard/Dashboard.styles.tsx]: fiquei sem criatividade para os nomes

File: src/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { PlayGround, Operations, NavBar, Container, Sidebar } from './Dashboard.styles';
+
+const Dashboard = () => {
+  return (
+    <Container>
```
Comment (davidcostadev) [inline src/pages/Dashboard/index.tsx:6]: Container -> Page

File: src/pages/Dashboard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <Sidebar />
+      <PlayGround>
```
Comment (davidcostadev) [inline src/pages/Dashboard/index.tsx:8]: PlayGround -> Wrapper

File: src/pages/Dashboard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        <NavBar />
+        <Operations />
```
Comment (davidcostadev) [inline src/pages/Dashboard/index.tsx:10]: Operations -> Content

**Review by davidcostadev — CHANGES_REQUESTED** [review]

**Review by davidcostadev — APPROVED** [review]

**Pull request merged**